server.php

```diff
@@ -17,6 +17,38 @@ function show_valet_404()
 }
 
 /**
+ * http://stackoverflow.com/a/10473026/2098861
+ *
+ * @param $haystack
+ * @param $needle
+ *
+ * @return bool
+ */
+function ends_with($haystack, $needle)
+{
+    // search forward starting from end minus needle length characters
+    return $needle === '' || (($temp = strlen($haystack) - strlen($needle)) >= 0 && strpos($haystack, $needle, $temp) !== false);
+}
+
+/**
+ * @param $domain string Domain to filter
+ *
+ * @return string Filter domain (without xip.io feature)
+ */
+function support_xip_io($domain)
+{
+    if (ends_with($domain, ".xip.io")) {
```

Comment:
Should I rename also `support_xip_io` to `valet_support_xip_io` (or something else that include `valet` word) ?